inexor/shared/cube_vector.hpp

```diff
@@ -67,7 +67,7 @@ template <class T, int MINSIZE = 8> struct vector
     /// Add new index to vector
     /// @brief Pushs a new element at the end of the vector. Allocates memory automaticly if neccesary.
     /// @param x Element which will be added at the end
-    /// @sideeffects May allocates a lot of memory without your notice
+    /// @note May allocates a lot of memory without your notice
```

Comment:
Thanks for letting me know, the reason why I replaced it with the "@note" command is because I couldn't find the "@sideeffect" command on their doxygen website. I did change it to the right spelling a while back but doxygen flagged it as a warning.